metricbeat/module/logstash/node/_meta/fields.yml

```diff
@@ -22,6 +26,8 @@
           description: >
             Version
         - name: pid
-          type: long
+          type: alias
+          path: process.pid
+          migration: true
```

Comment:
Hmmm wait, here it says you **are** aliasing it, though :-) Is the data.json file out of sync, then?

Reply:
I _guess_ it is, because the golang parsing code that generates that file is still placing the `pid` in two places. What is said here in `fields.yml` will become accurate once I place the `pid` only at the top level.